src/theme/index.ts

```diff
@@ -16,17 +52,18 @@ const Theme = {
     },
     background: {
       elevation_n1: '#0C0C0C',
-      elevation0: '#12151E',
-      elevation1: '#1D2032', // deprecated?
-      elevation_1: '#070A13',
-      elevation2: '#272A44',
-      elevation3: '#303354',
-      elevation6: '#4C5187',
-      elevation6_600: 'rgba(76, 81, 135, 0.4)',
-      elevation6_800: 'rgba(76, 81, 135, 0.2)',
-      elevation8: '#7E89AB',
-      elevation9: 'rgba(76, 81, 135, 0.2)',
-      elevation10: 'rgba(76, 81, 135, 0.35)',
+      elevation0: '#181818',
+      elevation1: '#1E2024',
+      elevation_1: '#1E2024', // deprecated
+      elevation2: '#24282F',
+      elevation3: '#2A2F39',
+      elevation5: '#303643',
+      elevation6: '#4C525F',
+      elevation6_600: 'rgba(76, 82, 95, 0.4)',
+      elevation6_800: 'rgba(76, 82, 95, 0.2)',
+      elevation8: '#7A7688',
+      elevation9: 'rgba(76, 81, 135, 0.2)', // deprecated
+      elevation10: 'rgba(76, 81, 135, 0.35)', // deprecated
```

Comment:
Were these supposed to change in this PR? Looks like you added the new ones above; is the plan not to migrate from these to those but keep these with the old values for now? Just checking because it seems like a potentially riskyt change 😅

Reply:
https://github.com/secretkeylabs/xverse-web-extension/pull/588